extensions/amp-list/0.1/amp-list.js

```diff
@@ -366,7 +366,7 @@ export class AmpList extends AMP.BaseElement {
     };
 
     const src = mutations['src'];
-    const state = /** @type {!JsonObject} */ (mutations)['state'];
+    const {state} = mutations;
```

Comment:
Will this `state` have its name mangled? Eg, we want `const s = mutations['state']`. But if it does `const s = mutations['s']`, then we have a bug.

Reply:
good point. i don't know why this is not caught by the checker. I'll double check

Reply:
<img width="515" alt="Screen Shot 2019-09-19 at 10 05 21 PM" src="https://user-images.githubusercontent.com/354746/65300820-a16e1900-db29-11e9-9880-911d5f4efd81.png">

checked and it doesn't get mangled. but i actually expected it to be. ill do some testing and file it against the CC team since then that should be a violation of the `@dict` type

Reply:
Its shape might match an unrelated extern. Though looks like this should still be changed to be safe.